### DISCUSSION TITLE
Color 에셋 오타

### DISCUSSION THREAD

**Discussion by KimDaeSeong8721** [DeveloperAcademy-POSTECH/toGather#54] [General]
<img width="477" alt="image" src="https://user-images.githubusercontent.com/69894461/173608966-6c808fa4-f8bf-424c-ade6-5430bdbf2b4b.png">

BasicBlack -> basicBlack으로 바꾸는게 좋을 거 같습니다!

**Comment by yudonlee** [comment]
@KimDaeSeong8721 가 오타수정 후 Merge
